website/dbt-global-variables.js

```diff
@@ -16,5 +16,17 @@ exports.dbtVariables = {
         "version": "0.21"
       }
     ]
-  } 
+  },
+    dbt: {
+    name: "dbt"
```

Comment:
This one feels optimistic to expect people to reference everywhere going forward, and has a very low likelihood of changing. The juice might not be worth the squeeze?

Reply:
> The juice might not be worth the squeeze?

LOL Yea I was just throwing a few in there. I was considering `dbt Labs` when we're using our official voice. And `dbt` to mean alll of the products, but I can keep it simple for now.